pkg/target/types.go

```diff
@@ -40,9 +40,10 @@ type Resource struct {
 type Application struct {
 	ObjectMeta
 
-	CPU    Resource
-	Memory Resource
-	QPS    Resource //This field is copied from Container.QPS
+	CPU          Resource
+	Memory       Resource
+	QPS          Resource  //This field is copied from Container.QPS
+	ResponseTime *Resource //This field is copied from Container.ResponseTime
```

Comment:
We can make all of them to be pointer of struct, or struct. But not some are struct, some is pointer.